azkaban-common/src/main/java/azkaban/project/validator/ValidatorManager.java

```diff
@@ -28,10 +28,9 @@
   Map<String, ValidationReport> validate(Project project, File projectDir);
 
   /**
-   * The ValidatorManager should have a default validator which checks for the most essential
-   * components of a project. The ValidatorManager should always load the default validator. This
-   * method returns the default validator of this ValidatorManager.
+   * ValidatorManager will not have any default validator.
    */
+  @Deprecated
   ProjectValidator getDefaultValidator();
```

Comment:
Found one internal team is using this method. Will coordinate with them about the migration solution.

Reply:
Good find!

Reply:
Already discussed with the team and they agree to use the old azkaban common module jar while we are developing new features. When the Flow 2.0 DSL change is rolled out, they will migrate to the new implementation or we provide some external API to them depending on the number of use cases then.